src/idealized_setup.jl

```diff
@@ -83,7 +84,7 @@ function idealized_setup(arch;
                         settings...)
 
     if isforced(model)
-        set!(model, T = Tᵢ) 
+        set!(model, v = vᶠ, T = Tᵢ) 
     else
         set!(model, u = uᵢ, v = vᵢ, T = Tᵢ)
```

Comment:
If it is not forced we don't need to add this velocity as well?

Reply:
Nice catch! I will add this as well.